rcon/player_history.py

```diff
@@ -227,6 +231,31 @@ def ban_if_blacklisted(rcon, steam_id_64, name):
             )
 
 
+def ban_if_has_vac_bans(rcon, steam_id_64, name):
+    try:
+        autoban_threshold = int(AUTOBAN_TRIGGER_THRESHOLD)
+    except ValueError: # No (proper) value is given
+        return
+
+    with enter_session() as sess:
+        player = get_player(sess, steam_id_64)
+
+        if not player:
+            logger.error("Can't check VAC history, player not found %s", steam_id_64)
+            return
+
+        bans = get_player_bans(steam_id_64)
+        if bans == {}: return # Player has no bans
+        last_ban = bans['DaysSinceLastBan']
+        if int(last_ban) <= autoban_threshold:
+            reason = f"VAC ban history ({str(last_ban)} days ago)"
```

Comment:
As mentionned in the issue it'd be better to make that configurable. Not everyone wants english :)

Reply:
The message for getting banned by the blacklist wasn't configurable either. Should I make that configurable as well then?

Reply:
Hmm, yes it is? It's a parameter from the GUI, then it get stored in the DB :) Unless you're talking about something else.
The one for VACs however should be from the `.env` as this is fully automatic. We could set it from the GUI but I think it's overkill TBH and too much extra work.